src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -219,12 +219,12 @@ uct_rc_mlx5_iface_common_am_handler(uct_rc_mlx5_iface_common_t *iface,
 }
 
 static UCS_F_ALWAYS_INLINE uint8_t
-uct_rc_mlx5_ep_fm(uct_rc_mlx5_iface_common_t *iface, uct_ib_mlx5_txwq_t *txwq)
+uct_rc_mlx5_ep_fm(uct_rc_mlx5_iface_common_t *iface, uct_ib_mlx5_txwq_t *txwq,
```

Comment:
rename to: uct_rc_mlx5_ep_fm_cq_update
because the function of this wrapper is to always add CQ_UPDATE flag